web/styles/settings.css

```diff
@@ -809,10 +816,13 @@ input[type="checkbox"] {
     margin: 20px 0 0;
 }
 
+#bot-edit-form,
```

Comment:
Do we want to increase width of bot owner dropdown?

![image](https://github.com/user-attachments/assets/80de5fb9-6dd9-4944-abdc-c63d817cc427)

Reply:
It displays the names with enough width, so I feel like it's fine to keep it as is. But if @alya (or Tim) wants it full width I'm happy to update it.

Reply:
I don't want to re-test right now, did the PR change this 2-line behavior?

![Screenshot 2025-02-05 at 12 49 11](https://github.com/user-attachments/assets/00133ac8-318b-496b-be52-6f37cad3d530)

Reply:
Long enough to fit the name seems OK, but the 2 lines are a bit silly when there's plenty of horizontal space.

Reply:
Oh shoot sorry I really thought I'd tested the longer names but I guess I didn't. I'll just make the dropdown full width then.

```diff
diff --git a/web/styles/modal.css b/web/styles/modal.css
index 3adec73cb1..933acdffa6 100644
--- a/web/styles/modal.css
+++ b/web/styles/modal.css
@@ -428,8 +428,10 @@
     width: fit-content;
 }
 
-.overlay-content .dropdown-widget-button {
-    width: var(--modal-input-width);
+.modal__body, .overlay-content {
+    .dropdown-widget-button, .dropdown_widget_with_label_wrapper {
+        width: var(--modal-input-width);
+    }
 }
 
 .modal_password_input,
```